python/compiler/txn2mlir.py

```diff
@@ -65,7 +65,7 @@ def parse_txn(data, verbose=False):
                 _, addr, size = struct.unpack("III", data[i + 4 : i + 16])
                 print_log(f"addr: {addr:#x}")
                 print_log(f"size: {size}")
-                operations.append((opc, addr, data[i + 16 : i + size]))
+                operations.append((opc, addr, data[i + 16 : i + size - 16]))
```

Comment:
What's the reason for dropping the last 16 data? If we do that, the control packets being generated will come with missing data, and fail (I tested locally with this e2e [test](https://github.com/Xilinx/mlir-aie/pull/1739/files#diff-02a23a98f065c1c6af63f615f8bb801d435144da9d14b0f2ea75fb79255ca298)).

Reply:
I reverted this in this commit: https://github.com/Xilinx/mlir-aie/pull/1739/commits/5310c72164e145c2eeff83a092d4b2f91e901f1c, in order to try to get the PR to work.

Reply:
hmm, that change was unintentional. Probably a merge mistake by me since I see you already fixed it once.  You are correct though, the end of the data should be `i + 16 + size - 16 = i + size`